### PR TITLE
[WIP] Bot based on upstream

### PR DESCRIPTION
**Pull request by gundalow**
Diff against `866a73b2c56e4328cb749e42086d08dfe5478f2c`

### CODE REVIEW

File: ansibullbot/triagers/ansible.py [hunk 1]
```diff
@@ -821,6 +820,8 @@ def create_actions(self, iw, actions):
                 if 'owner_pr' in iw.labels:
                     actions.unlabel.append('owner_pr')
 
+        """
+        FORK: PRs outside of ansible/ansible are expected to be community_review so we don't need to add these.
```
[the remark this answers is not in the document]
Comment (mkrizek) [reply ansibullbot/triagers/ansible.py:824]: FWIW I plan to remove the "support" functionality from ansibot in hopefully near future as everything in ansible/ansible is owned by the core team. I would say generally it is not needed in the collection bot either.

**Comment by dmsimard** [comment]
https://github.com/ansible-community/collection_bot/pull/27/commits/e20674e7973299a7a03cf74bf8f3d603c927ac46 was superseded by upstream in https://github.com/ansible/ansibullbot/pull/1579

**Comment by dmsimard** [comment]
https://github.com/ansible-community/collection_bot/pull/27/commits/addbb899a93ba93c48577d7c9a9ff1a60aa33a73 was superseded by upstream in https://github.com/ansible/ansibullbot/pull/1557

**Comment by dmsimard** [comment]
I've proposed https://github.com/ansible-community/collection_bot/pull/27/commits/6f74e0875bec06b9f886a369724c9b07373f42c6 upstream in https://github.com/ansible/ansibullbot/pull/1581

**Comment by dmsimard** [comment]
Closing in favor of https://github.com/ansible-community/collection_bot/pull/28 (to maintain history without force pushing)

**Pull request closed without merging**